### PR TITLE
fix UI: Confusing data about query on table details page. #8303

### LINKED ISSUE #8303: UI: Confusing data about query on table details page.
**Affected module**
Does it impact the UI, backend or Ingestion Framework?
UI

**Describe the bug**
On table details page, the 'Queries Usage Count' is shown as ' {number of usage} Queries' but we show only unique queries in the 'Queries' tab which is confusing for user. Need to update the message to state clearly what the count is about.

**To Reproduce**

Screenshots or steps to reproduce
Screenshot 2022-10-21 at 5 31 06 PM

**Expected behavior**

**Version:**
 - Python version:
 - OpenMetadata version: [e.g. 0.8]
 - OpenMetadata Ingestion package version: [e.g. `openmetadata-ingestion[docker]==XYZ`]

**Additional context**

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes:

Fixes #8303

UI: Confusing data about query on table details page. #8303
- Added ES count call to query to get query count and show to database details page

image
image

#
### Type of change:

- [ ] Bug fix
- [x] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/common/QueryCount/QueryCount.test.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+}));
+
+describe('QueryCount test', () => {
+  it('component should render', async () => {
+    render(<QueryCount tableId="testID" />);
+
+    expect(
+      await screen.findByText('10 label.query-plural')
+    ).toBeInTheDocument();
+  });
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/common/QueryCount/QueryCount.test.tsx:32]: Can you also add negative cases
1. No queries
2. Loading
3. API failure?

**Review by chirag-madlani — APPROVED** [review]
Awesome @ShaileshParmar11 🚀

File: openmetadata-ui/src/main/resources/ui/src/components/common/QueryCount/QueryCount.test.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/components/common/QueryCount/QueryCount.test.tsx:32]: Yes, I will add that after code freeze

**Review by chirag-madlani — APPROVED** [review]
Glossary failure will be covered here: https://github.com/open-metadata/OpenMetadata/pull/11027

**Pull request merged**